test/unit-and-integration/createProvider.test.ts

```diff
@@ -8,28 +8,20 @@ import userEvent from '@testing-library/user-event';
 globalThis.alert = () => {};
 
 describe('CreateProvider component', () => {
-  const mockCancelAction = vi.fn();
```

Comment:
cancelActionsand mockCancelActions all went away because now you just go back to root, as a link.